From 4adf53555fc15743bd417f74b1b59610c21fa96d Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Thu, 20 Oct 2016 16:47:33 -0500
Subject: [PATCH 0791/1164] Revert "CLOUDERA-BUILD: CDH-45436: Include
 hive-exec-core in packaging in cdh5.10"

This patch is causing problems when connecting to HS2.

This reverts commit aac2e55d061d0d597260381046e4fd517f025175.

Change-Id: Id8a844cdc32300db4f8ba633b407896ed72d0efd
---
 packaging/pom.xml |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/packaging/pom.xml b/packaging/pom.xml
index 47cf028..7a30e06 100644
--- a/packaging/pom.xml
+++ b/packaging/pom.xml
@@ -153,12 +153,6 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hive</groupId>
-      <artifactId>hive-exec</artifactId>
-      <version>${project.version}</version>
-      <classifier>core</classifier>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.hive</groupId>
       <artifactId>hive-service</artifactId>
       <version>${project.version}</version>
     </dependency>
-- 
1.7.9.5

